products/bash/setup-demos.sh

```diff
@@ -155,16 +162,58 @@ function merge_addon {
   echo ${ADDON_ARRAY_JSON}
 }
 
+function update_status() {
```

Comment:
Maybe rename this to "add_condition", derive the timestamp, just require the following as parameters:
```yaml
      message: 'Event Streams cluster is being deployed.'
      reason: 'Creating'
      status: 'True'
      type: 'Pending'
```